src/views/forms/AccountManagementForms/SourceSelectMenu.tsx

```diff
@@ -66,11 +70,26 @@ export const SourceSelectMenu = ({
         onSelect(chain.type, 'chain');
       },
       slotBefore: <Styled.Img src={chain.iconUrl} alt="" />,
+      [lowestFeesDecoratorProp]: (
+        <Styled.Text>
+          {stringGetter({
+            key: STRING_KEYS.LOWEST_FEES_WITH_USDC,
+            params: {
+              LOWEST_FEES_HIGHLIGHT_TEXT: (
+                <Styled.GreenHighlight>
+                  {stringGetter({ key: STRING_KEYS.LOWEST_FEES_HIGHLIGHT_TEXT })}
+                </Styled.GreenHighlight>
+              ),
+            },
+          })}
+        </Styled.Text>
+      ),
     }))
     .filter(
       (chain) =>
         type === TransferType.deposit || !!cctpTokenssByChainId[chain.value] || !CCTPWithdrawalOnly
-    );
+    )
+    .sort((chain) => (!!cctpTokenssByChainId[chain.value] ? -1 : 1));
```

Comment:
nit: looks like I made a typo for `cctpTokenssByChainId` 🤡

Reply:
!! haha i thought i fixed them all in my previous PR, looks like there was still some more 👀